backend/app/categories/controller.py

```diff
@@ -17,9 +18,23 @@
 async def get_category_route(category_id: str,
                              db: AsyncIOMotorClient = Depends(get_database)) -> ItemsInResponse:
     """Get the items under a given category"""
-    # TODO add fetching from category
-    return ItemsInResponse()
-
+        
+    if category_id not in CategoryEnum.__members__:
+        return ItemsInResponse(success=False, error=[f"Invalid category {category_id}"])
+    
+    loader = None
+    if category_id == CategoryEnum.movies:
+        loader = Movie
+    elif category_id == CategoryEnum.anime:
+        loader = Anime
+    elif category_id == CategoryEnum.music:
+        loader = Music
+    elif category_id == CategoryEnum.shows:
+        loader = Show
+    elif category_id == CategoryEnum.books:
+        loader = Book
+    items = [loader(**item) async for item in db[category_id]["data"].find()]
+    return ItemsInResponse(data=items)
```

Comment:
These changes shouldn't appear in your PR, you are doing something wrong

Reply:
ok i am rebasing and pushing again

Reply:
done